src/components/Header.js

```diff
@@ -0,0 +1,55 @@
+import { Navbar, Container, Nav, Link } from "react-bootstrap";
+import Parse from "parse";
+import { useState } from "react";
+import { useEffect } from "react";
+
+export function Header() {
+  const [loggedIn, setLoggedIn] = useState();
+  const [currentTab, setCurrentTab] = useState();
+
+  useEffect(() => {
+  if (Parse.User.current( )) setLoggedIn(true);
+  setCurrentTab(window.location.pathname);
+}, []);
+
+  function handleLogoutAttempt(e) {
+      alert("You have logged out");
+      setLoggedIn(false);
+      Parse.User.logOut();
+      window.location.href="/";
```

Comment:
Sure you want to setLoggedIn before Parse.user.logOut?
Maybe the parse logout returns an unforeseen error, and since it is asynchronous, your system will be out of sync.
This could be a non-issue in this case, but it is something you should consider always when using APIs.
Good rule is to await the API call, and then do your own thing. Catch in case of error and make sure to separate "on success" logic from "on failure" logic.